cmd/neofs-node/config.go

```diff
@@ -87,12 +87,16 @@ const (
 	cfgAccountingFee      = "accounting.fee"
 
 	// config keys for cfgNetmap
-	cfgNetmapContract = "netmap.scripthash"
-	cfgNetmapFee      = "netmap.fee"
+	cfgNetmapContract          = "netmap.scripthash"
+	cfgNetmapFee               = "netmap.fee"
+	cfgNetmapWorkerPoolEnabled = "netmap.async_worker.enabled"
```

Comment:
maybe it is better to add `notification` word here

Reply:
As for me, it is okay as it is. Don't mind to share this pool for something else related to contract.

Reply:
ok. keep that version